deploy/complete/helm-chart/mushop/charts/catalogue/config/atp.init.sql

```diff
@@ -51,89 +51,6 @@ BEGIN
 		END IF;
 	END;
 
-	-- products Table Creation
-	DECLARE
-		tableExists INTEGER;
-		tableName VARCHAR2 (20) := 'PRODUCTS';
-	BEGIN
-		SELECT COUNT(*) 
-		INTO tableExists 
-		FROM DBA_TABLES 
-		WHERE owner = '&1'
-		AND table_name = tableName;
-		DBMS_OUTPUT.PUT_LINE ('** Table creationg steps - &_DATE');
-		IF tableExists = 0 THEN
-			DBMS_OUTPUT.PUT_LINE ('Creating Table ' || tableName || '...' );
-			EXECUTE IMMEDIATE 'CREATE TABLE &1..' || tableName || ' (
-				sku VARCHAR2(20) NOT NULL,
```

Comment:
Tables are created now by flyway so there is no need to have this code here anymore.